hyper/hyper/save_test.go

```diff
@@ -0,0 +1,58 @@
+package hyper_test
+
+import (
+	"testing"
+
+	"github.com/isard-vdi/isard/hyper/hyper"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"libvirt.org/libvirt-go"
+)
+
+func TestSave(t *testing.T) {
+	require := require.New(t)
+	assert := assert.New(t)
+
+	cases := map[string]struct {
+		PrepareDesktop func(h *hyper.Hyper) *libvirt.Domain
+		ExpectedErr    string
+	}{
+		"save the desktop correctly": {
```

Comment:
Should check if the saved file is valid

Reply:
Can't we assume the file is 'correct' if the save function didn't return an error?

Reply:
ok, done

Reply:
Where is it done?

Reply:
I assumed that if desktop.Save is correct the file will be correct. We should rely on libvirt function.